jmxtrans-core/src/main/java/com/googlecode/jmxtrans/connections/JMXConnection.java

```diff
@@ -30,31 +31,81 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.ThreadSafe;
+import javax.management.AttributeChangeNotification;
+import javax.management.InstanceNotFoundException;
 import javax.management.MBeanServerConnection;
+import javax.management.Notification;
+import javax.management.NotificationFilter;
+import javax.management.ObjectName;
 import javax.management.remote.JMXConnector;
 import java.io.Closeable;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.IdentityHashMap;
+import java.util.List;
 
 @ToString
 @ThreadSafe
 public class JMXConnection implements Closeable {
-	@Nullable private final JMXConnector connector;
-	@Nonnull @Getter private final MBeanServerConnection mBeanServerConnection;
+	@Nullable
```

Comment:
It would be nice to not reformat code that you don't touch. It makes reviewing it much easier...

Reply:
Sorry was a mistake and yes makes reviewing a lot easier ;) I'll double check next time.

Reply:
No problem, that's really a minor issue...